### PR TITLE
Crypto appendix: allow HMAC-SHA-1

### PR DESCRIPTION
**Pull request by randomstuff**

### CODE REVIEW

**Comment by tghosth** [comment]
Hi @randomstuff did you discuss this with @danielcuthbert? Would want to get his sign-off before merging this.

**Comment by tghosth** [comment]
Is there a related open issue?

**Review by danielcuthbert — COMMENTED** [review]
I'm ok with this addition. Good to be specific about using SHA-1 for HMAC only and we can monitor this if anything changes

**Review by tghosth — APPROVED** [review]

**Pull request merged**
